lib/GenericsAPI/Utils/MatrixUtil.py

```diff
@@ -444,101 +445,65 @@ def _generate_rarefy_visualization_content(self, output_directory,
         return tab_def_content + tab_content
 
     def _generate_trans_visualization_content(self, output_directory,
-                                              filtered_matrix_dir, relative_abundance_matrix_dir,
-                                              standardize_matrix_dir,
-                                              ratio_transformed_matrix_dir, transformed_matrix_df):
+                                              operations, heatmap_html_dir_l, transformed_matrix_df):
         row_data_summary = transformed_matrix_df.T.describe().to_string()
         col_data_summary = transformed_matrix_df.describe().to_string()
 
         tab_def_content = ''
         tab_content = ''
+        
+        op_2_name = {
+            'abundance_filtering': 'Filtered',
+            'standardization': 'Standardized',
+            'ratio_transformation': 'Log Ratio Transformed',
+            'relative_abundance': 'Relative Abundance',
+            'logit': 'Logit',
+            'sqrt': 'Square Root',
+            'log': 'Log',
+        }
 
-        viewer_name = 'data_summary'
+        ## Start tabs ##
         tab_def_content += '''\n<div class="tab">\n'''
+
+        ## Operations tabs ##
+        for i, (op, heatmap_html_dir) in enumerate(zip(operations, heatmap_html_dir_l)):
```

Comment:
Yeah I wanted to make the stats summary tab last. I thought it was confusing to have it first, because the order of the tabs was in chronological order otherwise. What do you think?

Reply:
👍  sure. Let's keep it this way for now.